semgrep/tests/e2e/snapshots/test_baseline/test_all_intersect/baseline_error.txt

```diff
@@ -1,6 +1,13 @@
-Running baseline scan with base set to: baseline-commit
+Scanning 1 file.
+  Current version has 1 finding.
 
-skipped: all .gitignored files
-for a detailed list of skipped files, run semgrep with the --verbose flag
+Switching repository to baseline commit 'baseline-commit'.
+  Will report findings introduced by these commits:
+    * 1c6ece2 second
 
-ran 1 rules on 1 files: 0 findings
+Scanning 1 file.
+
+Some files were skipped.
+  Scan was limited to files changed since baseline commit.
```

Comment:
Because when we get the list of files to scan from `git diff`, we save the effort of walking through the filesystem, and thus don't even know what other files are in the directory.